portal/src/it/resources/application.conf

```diff
@@ -51,3 +51,9 @@ database {
     collection = ${?CMWL_TEST_MONGO_COLLECTION}
   }
 }
+
+cromwell-backend {
+    host = ${?CMWL_AWS_HOST}
```

Comment:
Why are we using "AWS" segment here? It could be tricky. Maybe it would be better to take "CROMWELL"? Doesn't sound really good with prefix, but more obvious.